utm-domains/utm-domain-commons.yaml

```diff
@@ -1319,12 +1323,12 @@ definitions:
           field is required for all message_types except OTHER_SEE_FREE_TEXT.
           1. UNPLANNED_LANDING
             This operation had to land earlier than expected and/or at a
-            location other than was originally planned. Must be sent with
+            location other than was originally planned. MUST be sent with
```

Comment:
Consider emphasizing the full **MUST** statements in markdown?  Something like:

`**MUST be sent ...**  ` 

so that it renders as:
 
 **MUST be sent ...**

This is just a suggestion for consideration.  Use at your discretion.